time/src/test/java/io/spine/time/string/TimestampStringifierShould.java

```diff
@@ -31,6 +33,9 @@
  */
 public class TimestampStringifierShould extends AbstractStringifierTest<Timestamp> {
 
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
```

Comment:
As long as we agree on the fact that this rule is deprecated, please consider using `assertThrows` in the test code below instead.